recipes/remix/app/routes/edit.tsx

```diff
@@ -0,0 +1,76 @@
+import { Puck, type Data } from "@measured/puck";
+import styles from "@measured/puck/dist/index.css";
```

Comment:
New format for this now!

```suggestion
import styles from "@measured/puck/puck.css";
```